src/app/components/cds/Sandbox.js

```diff
@@ -1935,6 +1950,38 @@ const SandboxComponent = ({ admin }) => {
       { (!selectedCategory || selectedCategory === 'charts') &&
         <section>
           <h6>Charts</h6>
+          <div className={styles.componentWrapper}>
+            <div className={styles.componentControls}>
+              <div className={cx('typography-subtitle2', [styles.componentName])}>
+                StackedBarChartWidget
+                <a
+                  className={styles.figmaLink}
+                  href="https://www.figma.com/design/82Go6q0krKApn1L8EQ2joj/Dashboard?node-id=186-5696&node-type=symbol"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                  title="Figma Designs"
+                >
+                  <FigmaColorLogo />
+                </a>
+                <ul>
```

Comment:
@amoedoamorim here is a petty change, but if you move the `<ul>` outside of the `styles.componentName` div, then the switch will be "below" the title and match the other sandbox sections